recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "0.6.1" %}
+{% set version = "0.9.0" %}
 
 package:
   name: torchdata
   version: {{ version }}
 
 source:
   url: https://github.com/pytorch/data/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: c596db251c5e6550db3f00e4308ee7112585cca4d6a1c82a433478fd86693257
+  sha256: b547bbe848ad813cc5365fe0bb02051150bec6c7c4ee7bffd6b6d3d7bdeddd75
 
 build:
   number: 0
   # pytorch 2.0.x not available on ppc64le
-  skip: true  # [py<37 or ppc64le]
+  skip: true  # [py<39 or py>311 or s390x]
```

Comment:
What is the reason for skipping `py>311`?
I see it could be portalocker. Could we build portalocker instead?

Reply:
Yes, it looks like its due to:

```
portalocker-2.3.0-py38hca03da5_0 requires python >=3.8,<3.9.0a0, but none of the providers can be installed
```

Would building `portalocker` be a better solution?

Reply:
Yes